### ISSUE TITLE
Add Reverb effect

### ISSUE THREAD

**Issue by Sumsun93** [saltminede/saltychat-docs#14]
Hi !

Is it possible to add a reverb effect with an intensity option ?

**Comment by BlackFlash5** [comment]
Isn't reverb just an echo?
We already got an echo effect, but it isn't controllable yet.

**Comment by Sumsun93** [comment]
No, isn't really the same. I've record the difference : https://voca.ro/fBF10Q1xNKU

**Comment by BlackFlash5** [comment]
What is the use case for this anyways?
If it's a edge case for only a hand full of people, then it will most likely take too much time and has no real benefits.

**Comment by Sumsun93** [comment]
For example, in native GTAO vocal, we have an reverb when speaking in interior.
And the intensity of effect is increased by the size of interior (hangar, caves, etc..)

Echo effect is more useless than reverb effect... it's usefull only in big big big caves or on top of chiliad mountain

**Comment by BlackFlash5** [comment]
Echo was primarily for tunnels.
And tbh reverb is more useless, because i'd bet more then 90% of the people wouldn't care at all about this in interiors.
I'll add it to the bottom of my list and have to somehow figure out how you even calculate something like that, since i'm not a sound engineer.
